examples/individual_properties/main.rs

```diff
@@ -118,10 +118,10 @@ async fn main() -> eyre::Result<()> {
         // Send in a loop the change of the property "name" of sensor 1
         loop {
             client_cl
-                .send(
+                .send_individual(
```

Comment:
Why don't we use the new property api?
```suggestion
                .set_property(
```